### ISSUE TITLE
Accessibility 89!

### ISSUE THREAD

**Issue by frannyfra** [fac18/week2-CEFG-To-do#20]
👍 👍 

Well done team. Light house audit says you have got a 89% , which is a good number.

There are some minor issue:
"Buttons do not have an accessible name
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers."

I think it refer to the button you have created in your dom. js on line 37:
 ``
markTodoButtonNode.setAttribute("class", "mark-button");

``
You might want to replace the class with the name so it will be accessible!
Will add additional link later 😄
